app/templates/public/exhibition/view.hbs

```diff
@@ -22,10 +24,10 @@
             </div>
         {{/if}}
         <br/><br/>
-        <h1 class="ui centered header no margin">
-            <img class="ui small image m-0 mb-1" src="{{or this.model.exhibitor.logoUrl '/images/placeholders/Other.jpg'}}" alt="Logo">
-            {{this.model.exhibitor.name}}
-        </h1>
+        <div class="d-flex items-center content-center">
+            <img class="ui small image m-0 mb-1" src="{{or this.model.exhibitor.logoUrl '/images/placeholders/Other.jpg'}}" alt="Logo" style="height:60px; width:auto; object-fit:contain;">
+            <h1 style="margin-top:0px; padding-top:0px;">{{this.model.exhibitor.name}}</h1>
```

Comment:
maybe go with height: 45px

Reply:
Mam They will look like this in case of 45 px. Please tell me if this looks fine.<br></br>
![Screenshot (1097)](https://user-images.githubusercontent.com/63748249/110827348-fb354680-82bb-11eb-9ee4-50688c83c9af.png)
<br></br>
![Screenshot (1099)](https://user-images.githubusercontent.com/63748249/110827353-fc667380-82bb-11eb-9655-a32e25264b7b.png)

Reply:
This currently is without padding, But I will surely add the padding while Pr.

Reply:
@MeghalBisht  Mam Could You Please confirm this.So that I could proceed with this PR.